packages/core/src/generateAppDeclarationFile.ts

```diff
@@ -66,6 +83,7 @@ export default function generateAppDeclarationFile(
   // console.log("---123----")
```

Comment:
console.log

Reply:
These are some test logs I use from time to time.